CONTRIBUTING.md

```diff
@@ -48,8 +45,14 @@ for any of the following plugins, we encourage you to submit it
    fetch from the master repository, not your clone, when running `git fetch`
    et al.)
 
-## Running the examples
 
+## Setting up tools
+
+We use a range of tooling script to do varies of things on CI. (testing, formatting, etc.)
+Your are likely to use some of the tooling script locally in your contributing journey.
```

Comment:
How about just: "There are scripts for many common tasks (testing, formatting, etc.) that will likely be useful in preparing a PR.". It seems odd to frame it in terms of CI first when that's not the use case for most readers.